app/api/stores/schemas.py

```diff
@@ -38,3 +38,12 @@ class StoreUpdate(BaseModel):
             regex=telephone_number_regex
         )
     ]
+
+
+class SearchItem(BaseModel):
+    item_id: UUID4
+    store_id: UUID4
+    name: constr(min_length=1, strip_whitespace=True)
+    price: int
+    inventory: int
+    photos: Optional[UUID4]
```

Comment:
```suggestion
    photo: Optional[UUID4]
```

只有一張照片，issue 打錯，有更新了